src/xref_runner.erl

```diff
@@ -85,7 +100,10 @@ check(Check, Config) ->
 
     FilteredResults = filter_xref_results(Check, Results),
 
-    [result_to_warning(Check, Result) || Result <- FilteredResults]
+    ResultToPrint =
+      [result_to_warning(Check, Result) || Result <- FilteredResults],
+    io:format("~p", [ResultToPrint]),
```

Comment:
This shouldn't be done here. This function should've been left as it was and the result should've been parsed and printed with a nicer format in the caller